test/cpp/jit/test_backend.cpp

```diff
@@ -1,81 +1,42 @@
+#include <gtest/gtest.h>
+#include <torch/csrc/jit/api/module.h>
 #include <torch/csrc/jit/backends/backend.h>
+#include <torch/csrc/jit/backends/backend_detail.h>
+#include <torch/csrc/jit/frontend/resolver.h>
```

Comment:
Why is this header needed? I don't see any of the types from the file used here.

Reply:
Removed.